CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fix the bug in the LIKE predicate which prevents using escaped backslash
```

Comment:
Maybe rephrase this a bit. "the bug in the LIKE predicate" sounds as if it's referring to one specific well-known bug.